src/commands/cluster-replicate.json

```
@@ -1,10 +1,10 @@
 {
     "REPLICATE": {
-        "summary": "Configure a node as replica of a primary node.",
+        "summary": "Configure a node as replica of a primary node or detaches replica from primary.",
         "complexity": "O(1)",
         "group": "cluster",
         "since": "3.0.0",
-        "arity": 3,
+        "arity": -4,
```

```suggestion
        "arity": -3,
```

-3 = 3 or more. `CLUSTER REPLICATE node-id` is 3.

---

Yes. Already figured it out.